CHANGELOG.md

```diff
@@ -2,13 +2,64 @@
 
 ## 5.0.0
 
-This version is compatible with SLDS 2.5.2 aka `Spring '18` 🌸
+This version is compatible with SLDS 2.5.2 aka `Spring'18` 🌸
 
-* _Tabs:_ **Breaking** Reimplement `Tabs`, `ControlledTabs`
-* _Tabs_: Support keyboard interactions for `Tabs`
-* _Button:_ **Breaking** Reimplement `Button`s, add `IconButton`, `StatefulButton`, `StatefulIconButton` components
-* _Modal:_ **Breaking** Reimplement `Modal`
-* _Modal:_ Trap focus inside `Modal`, support keyboard interactions
+### Modal
+
+[Storybook Docs](https://propertybase.github.io/react-lds/?selectedKind=Modal&selectedStory=Default)
+
+Modals have been reimplemented to require less code and support keyboard interactions:
+
+* Modals now trap focus when open and can be closed via `esc`-key
+* `Backdrop ` has been removed and is now implemented by `Modal`
+* `ModalHeader` has been removed. Setting a `title`, `tagline` or `onClose` function in `Modal` will now automatically render a header
+* `Modal` now requires an `id`. `ModalContent` will be automatically linked
+* `ModalFooter` now will automatically render a `cancel`-button when an `onClose` callback is set on `Modal`. You can set the label via `closeButtonLabel` or hide the button via `hideCloseButton`
+* If no `onClose` is set, the `Modal` is assumed to be uncloseable and no close buttons will be rendered
+* Modals now support setting the `transitionStyle` when opening and closing
+* `Prompt` has been updated to reflect the changes to `Modal`. A `Prompt` should be used if only one option is available
+
+### Tabs
+
+[Storybook Docs](https://propertybase.github.io/react-lds/?selectedKind=Tabs&selectedStory=Default)
+
+Tabs have been reimplented and now support keyboard interactions:
+
+* Added a `Tabs` component that renders n `Tab` children. Each `Tab` should have an `id`, `title` and `children`
+* Added a `ControlledTabs` component that does not handle events
+* Invisible tabs now render `null` instead of their content. You can change that behavior by setting `renderInvisibleTabs` to `true`
+* The Tab title can be rendered via a `renderTitle` function
+* Tabs can now be navigated via `arrow`-keys
+
+### Buttons
+
+[Storybook Docs: Button](https://propertybase.github.io/react-lds/?selectedKind=Button&selectedStory=Default) | [Storybook Docs: IconButton](https://propertybase.github.io/react-lds/?selectedKind=IconButton&selectedStory=Default)
+
+Buttons have been reimplemented to be understandable and require less boilerplate:
```

Comment:
What about:

> Buttons have been reimplemented to have a more consistent and declarative API. New shortcut props and good defaults require less boilerplate code:

Reply:
Love it. _...better defaults_, if they are actually good remains to be seen 😛